client/my-sites/checkout/checkout-thank-you/index.tsx

```diff
@@ -712,24 +712,8 @@ export class CheckoutThankYou extends Component<
 
 		// standard thanks page
 		return (
-			<Main
-				className={ classNames( 'checkout-thank-you', {
-					'is-redesign-v2': isRedesignV2( this.props ),
-				} ) }
-			>
+			<Main className="checkout-thank-you">
 				<PageViewTracker { ...this.getAnalyticsProperties() } title="Checkout Thank You" />
-				{ this.isDataLoaded() && isRedesignV2( this.props ) && (
```

Comment:
We no longer need the redesign checks here. All redesigned pages should render using the logic defined by these lines: https://github.com/Automattic/wp-calypso/pull/86271/files#diff-daac408bb350333e9fb8ca7c03233e75f03656e02a45c152e574227e6d253ee7R569-R610